web-animations-1/Overview.bs

```diff
@@ -822,6 +831,29 @@ in which they are created. Certain <a lt="animation class">classes of
 animations</a>, however, may provide alternative means of ordering animations
 (see [[#animation-classes]]).
 
+To convert a <dfn lt="time-based animation to a proportional animation">,
+time-based animation to a proportional animation</dfn>, effectively the
+[=timeline duration=] is divided up by the relative proportions of the
+specified times of the <a>animation</a>, <var>animation</var>. Concretely:
+
+1.  If the [=iteration duration=] is auto, then perform the following steps.
+
+    1.   Compute [=start delay=] and [=end delay=] to 0, as it is not possible
+         to mix time and proportions. Future versions may allow these properties
+         to be assigned percentages.
+
+1.  Let <var>total time</var> be equal to the result of evaluating
+    <code>|start delay| + |iteration count| * |iteration duration| + |end delay|</code>.
+
+1.  Compute [=start delay=] to be the result of evaluating
+    <code>|start delay| / |total time|</code>.
+
+1.  Compute [=iteration duration=] to be the result of evaluating
+    <code>|iteration duration| / |total time|</code>.
```

Comment:
Need to divide by  iteration count.

Reply:
I think it's correct as is, e.g. with start delay 2, iteration count 5, iteration duration 1, end delay 3:
total time is 2 + 5 * 1 + 3 = 10
iteration duration is calculated as 1 / 10 = 10%